### ISSUE TITLE
[Accessibility]: Non-code file view: File action buttons don't respond to keypress

### ISSUE THREAD

**Issue by umpox** [sourcegraph/sourcegraph#35853]
### Audit type

Keyboard navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/34432

### Problem description

<img width="494" alt="image" src="https://user-images.githubusercontent.com/9516420/169797697-6b5e09cf-c95f-4fb2-9891-02efd937cfea.png">

Pressing enter does not trigger these items (e.g. this one should link to GitHub, but instead dismisses the menu)

### Expected behavior

It should be possible to action these items using only a keyboard

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [ ] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [ ] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.
